CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## v0.13.0
+
+### Vision
+- Added video recording for custom `VideoSource` implementation
+- Updated Vision, Vision AR and Vision Safety to Android SDK 10
+- Added all required permissions to Vision SDK manifest.
+- Added selection of default 960x540 camera resolution or closest to 16x9 in Camera2VideoSourceImpl
```

Comment:
Nit : `Changed default resolution of Camera2VideoSourceImpl to 960*540`

Reply:
fixed